hbp_knowledge/receptors/djouma2006.bel

```diff
@@ -74,67 +74,117 @@ DEFINE ANNOTATION TextLocation  AS URL "https://arty.scai.fraunhofer.de/artifact
 SET Citation ={"PubMed","16820021"}
 
 SET Evidence ="Autoradiographic detection of dopamine D1 receptors by [125I]- SCH 23982 was fully displaceable by the agonist SKF-77434."
+SET Confidence = "High"
 complex(p(RGD:Drd1),a(MESH:"8-iodo-2,3,4,5-tetrahydro-3-methyl-5-phenyl-1H-3-benzazepine-7-ol"))
 complex(p(RGD:Drd1),a(HBP:"SK and F 77434"))
 a(HBP:"SK and F 77434") -| a(MESH:"8-iodo-2,3,4,5-tetrahydro-3-methyl-5-phenyl-1H-3-benzazepine-7-ol")
+UNSET Confidence
 
 SET Evidence ="Overall, no changes were observed in dopamine D1 receptor binding in the brain regions examined following social isolation and ⁄ or antalarmin treatment under these conditions"
+SET Confidence = "High"
 a(MESH:antalarmin) cnc act(p(RGD:Drd1))
+UNSET Confidence
 
 SET Evidence ="As can be seen from Fig. 1A, binding was dense throughout the NAcc, caudate-putamen (CPu), olfactory tubercle (OT) and substantia nigra, pars reticulata (SNr)."
+SET Confidence = "High"
 SET MeSHAnatomy ={"Nucleus Accumbens","Neostriatum","Olfactory Tubercle","Substantia Nigra","Pars Reticulata"}
 complex(p(RGD:Drd1),a(HBP:"SK and F 77434"))
+UNSET Confidence
 
 SET Evidence ="Autoradiographic binding of [125I]-NCQ 298 to dopamine D2 ⁄ 3 receptors was specific and fully displaceable by raclopride."
+SET Confidence = "Medium"
 complex(a(HBP:"[125I]-NCQ"),p(RGD:Drd1))
 complex(p(RGD:Drd1),p(MESH:Raclopride))
+composite(p(MESH:Raclopride),complex(p(RGD:Drd1),a(HBP:"[125I]-NCQ"))) -> complex(p(RGD:Drd1),p(MESH:Raclopride)) #because the evidence says that [125I]-NCQ was displaceable by raclopride
```

Comment:
composite would be that the result only happens if all components are present. As both chemicals are displaceable I woud just code it as two different processes where the result is the same

Reply:
I think I have done this mistake of complex and composite at many places. Were the previous comments doing the enough justice to the evidence?